azext_edge/tests/edge/orchestration/test_secretsync_int.py

```diff
@@ -147,6 +151,7 @@ def test_secretsync(secretsync_int_setup):
         kv_id=kv_id,
         mi_principal_id=mi_principal_id
     )
+    # note that since the role assignments are not applied, the secrets will not be synced
```

Comment:
Is this accurate? Secretsync enable should apply role assignments.

Reply:
updated the comment